modules/sdk/src/main/scala/org/tessellation/sdk/http/routes/ClusterRoutes.scala

```diff
@@ -73,6 +73,7 @@ final case class ClusterRoutes[F[_]: Async](
             case SessionDoesNotExist    => Conflict("Peer does not have an active session.")
             case CollateralNotSatisfied => Conflict("Collateral is not satisfied.")
             case NodeNotInCluster       => Conflict("Node is not part of the cluster.")
+            case InvalidNodeState       => Conflict("The node cannot be joined.")
```

Comment:
"The node cannot be joined." does not say why.
IMO it should be rather "Node has invalid state" or:
```scala
case InvalidNodeState(nodeState) => Conflict(s"Node cannot be joined to state=${nodeState}")
```

Reply:
This is no longer relevant.